src/api/app/views/webui/package/side_links/_create_badge.html.haml

```diff
@@ -11,9 +11,7 @@
         .modal-header
           %h5.modal-title#badge-modal-label
             Create Build Results Badge
-          %button.close{ type: 'button', 'data-dismiss': 'modal', 'aria-label': 'Close' }
-            %span{ 'aria-hidden': 'true' }
-              &times;
+          %button.btn.btn-close.float-end{ type: 'button', 'data-dismiss': 'modal', 'aria-label': 'Close' }
```

Comment:
```suggestion
          %button.btn.btn-close.float-end{ 'data-dismiss': 'modal', 'aria-label': 'Close' }
```
Not related to this change, but if I'm not mistaken we could get rid of the `type: 'button'` part, here and in a couple of places more.

Reply:
Forget it. This is needed.